### DISCUSSION TITLE
[Feature] <Request to add Chinese language support>

### DISCUSSION THREAD

**Discussion by YYTLZXD2327** [immich-app/immich#8039] [Feature Request]
### The feature

Description:
I would like to make a suggestion to this project to add Chinese language support. Here's what I think this change will benefit the project and how I'm willing to help:
Benefit:
Chinese is one of the most spoken languages in the world, providing great potential for the project to attract more Chinese users.
By adding Chinese support, you can increase the accessibility of your project, attract more potential users, and expand your project's reach.

Really begged

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by rjhenry** [comment]
Adding an additional feature request (from https://github.com/immich-app/immich/discussions/7148) is unlikely to get anywhere. Also note similar requests, https://github.com/immich-app/immich/discussions/2377, https://github.com/immich-app/immich/discussions/5817, and https://github.com/immich-app/immich/discussions/6383. As noted in https://github.com/immich-app/immich/discussions/6383, there's no translation support for the web yet. See https://github.com/immich-app/immich/discussions/1688.

Mobile app translations are already implemented - see [here](https://github.com/immich-app/immich/blob/main/mobile/assets/i18n/zh-CN.json) though I can't find documentation on how to use it.
